.github/workflows/linkchecker.yml

```diff
@@ -0,0 +1,33 @@
+name: Link checks
+
+on:
+  # Run manually by clicking a button in the UI
+  workflow_dispatch:
+  # Run every 8 hours
+  schedule:
+    - cron: '0 */8 * * *'
+
+env:
+  CONDA_ENV: test-env
+  GITHUB_ACTIONS: 'true'
+  OS_NAME: 'linux'
+  PYTHON_VERSION: 3.9
+
+jobs:
+  check-links:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 5
+          submodules: false
+      - name: Check that all tests succeeded
+        run: |
+          export BUILD_DIRECTORY="$GITHUB_WORKSPACE"
+          export TASK="check-links"
```

Comment:
Move to `env` section above.

Reply:
ha sure, I went back and forth on that one. pushed in https://github.com/microsoft/LightGBM/pull/3890/commits/64268d886e913b33ca60acba75f4d34745dbd15f